Homework/Hang_Nguyen/lesson_5/summary-about-ACID.md

```diff
@@ -0,0 +1,75 @@
+<b><h3>A Transaction</h3></b>
+
+*A transaction is a way for an application to group several reads and writes together into a logical unit.*
```

Comment:
can you give real-world examples for each bullet point? the most important point of a theory question is giving example